Test/Install-WhiskeyNuGetPackage.Tests.ps1

```diff
@@ -0,0 +1,133 @@
+& (Join-Path -Path $PSScriptRoot -ChildPath 'Initialize-WhiskeyTest.ps1' -Resolve)
+
+function Init
+{
+    $script:threwException = $false
+    $script:taskParameter = $null
+    $script:buildroot = $TestDrive.FullName
+    $script:result = $null
+}
+
+function WhenRunningNuGetInstall
+{
+    [CmdletBinding()]
+    param(
+        $Package,
+        $Version
+    )
+
+    $params = @{}
+    $params['Name'] = $Package
+    $params['Version'] = $Version
+    $params['DownloadRoot'] = $TestDrive.FullName
+
+    $Global:Error.Clear()
+    try
+    {
+        $script:result = Invoke-WhiskeyPrivateCommand -Name 'Install-WhiskeyNuGetPackage' -Parameter $params
+    }
+    catch
+    {
+    }
+}
+
+function ThenValidPackage
+{
+    param(

```

Comment:
Add parameters for the package name and version. Right now you're testing if any package is installed. We need to test that the specific package and version we're expecting were installed.